Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2628,3 +2628,47 @@ function Remove-CustomRole {
     }
 }
 
+<#
+    .Synopsis
+    This function allows the customer to restart HA on a cluster. This might be needed to clear HA alarms.
+    .EXAMPLE
+    Once the function is imported: Restart-ClusterHA -cluster_name "Cluster-1"
+#>
+function Restart-ClusterHA {
+    [AVSAttribute(30, UpdatesSDDC = $true)]
+    param(
+        [Parameter(Mandatory = $true,
+            HelpMessage = "What cluster to restart HA on.")]
+        [String]
+        $cluster_name
+    )
+
+    # Get the cluster object
+    try {
+        $cluster = Get-Cluster -Name $cluster_name -ErrorAction Stop
+    }
+    catch {
+        throw "Cluster $cluster_name not found."
+    }
+
+    $ha_status = ""
+
+    # Disable HA
+    $ha_status = ($cluster | Set-Cluster -HAEnabled:$false -Confirm:$false).HAEnabled
+
```

Comment:
Should there be a wait timer on this?  Does the set-cluster cmdlet wait for completion of disable call?

Reply:
HA getting disabled is not typically instant, so little concerned here on testing done.

Reply:
Timers added to changes.